src/components/tooltips/line-popover.js

```diff
@@ -0,0 +1,290 @@
+            }}
+            anchorOrigin={{
+                vertical: 'bottom',
+                horizontal: 'left',
+            }}
+            transformOrigin={{
+                vertical: 'top',
+                horizontal: 'right',
+            }}
+            onClose={handlePopoverClose}
+            open={Boolean(anchorEl)}
+            disableRestoreFocus
+        >
+            {lineInfo === null && (
+                <Box height={2} minWidth={100}>
+                    <LinearProgress />
+                </Box>
+            )}
+
+            {lineInfo !== null && (
+                <Grid
+                    container
+                    rowSpacing={2}
+                    direction={'column'}
+                    alignItems={'center'}
+                >
+                    <>
+                        <Grid item>
+                            <Typography variant="h6">{lineId}</Typography>
+                        </Grid>
+
+                        <Grid item>
+                            <TableContainer component={Paper}>
+                                <Table size={'small'}>
+                                    <TableHead>
+                                        <TableRow>
+                                            <TableCell>
+                                                {intl.formatMessage({
+                                                    id: 'Intensity',
+                                                })}
+                                            </TableCell>
+                                            <TableCell>
+                                                {checkValue(
+                                                    lineInfo.voltageLevelId1
+                                                )}
+                                            </TableCell>
+                                            <TableCell>
+                                                {checkValue(
+                                                    lineInfo.voltageLevelId2
+                                                )}
+                                            </TableCell>
+                                        </TableRow>
+                                    </TableHead>
+                                    <TableBody>
+                                        <TableRow>
+                                            <TableCell>
+                                                {intl.formatMessage({
+                                                    id: 'I_(A)',
+                                                })}
+                                            </TableCell>
+                                            <TableCell>
+                                                {loadFlowStatus ===
+                                                RunningStatus.SUCCEED
+                                                    ? checkValue(
+                                                          Math.round(
```

Comment:
If the rounded value is zero, then there is no displayed value at all, because checkValue filters it.

Reply:
fixed